studio/components/interfaces/Realtime/Inspector/EventsTable.tsx

```diff
@@ -0,0 +1,174 @@
+import { isEqual } from 'lodash'
+import { Loader2 } from 'lucide-react'
+import { Key, useEffect, useState } from 'react'
+import DataGrid, { RenderRowProps, Row } from 'react-data-grid'
+import { Button, IconBroadcast, IconDatabaseChanges, IconExternalLink, IconPresence, cn } from 'ui'
+
+import { useParams } from 'common'
+import { useRouter } from 'next/router'
+import LogSelection from './EventSelection'
+import { LogData } from './Events.types'
+import { ColumnRenderer } from './RealtimeEventColumnRenderer'
+
+export const isErrorLog = (l: LogData) => {
+  return l.event_message === 'SYSTEM' && l.metadata?.status === 'error'
+}
+
+interface EventsTableProps {
+  enabled: boolean
+  data?: LogData[]
+  showSendEvent: () => void
+}
+
+const NoResultAlert = ({
+  enabled,
+  showSendEvent,
+}: {
+  enabled: boolean
+  showSendEvent: () => void
+}) => {
+  const router = useRouter()
+  const { ref } = useParams()
+
+  return (
+    <div className="w-full max-w-md flex items-center flex-col">
+      {enabled ? <div>No Realtime events found</div> : null}
+      <div className="text-foreground-lighter">Realtime event logs will be shown here</div>
```

Comment:
When?
Maybe this should be the heading and the sub-heading should be a description or instruction. 

ie:

**Realtime event logs will be shown here** 
Choose a channel to get started showing logs.....